SECURITY.md

```diff
@@ -0,0 +1,47 @@
+# Security Policy
+
+Thank you for helping improve the security of the Mellanox network-operator project. We take security very seriously and appreciate your efforts to responsibly disclose any vulnerabilities you may discover.
+
+## Supported Versions
+
+We currently maintain and support security updates for the latest stable release of the network-operator. Please refer to the [release notes](./CHANGELOG.md) for details on the supported versions. If you are using an older release, we encourage you to upgrade to the latest secure version as soon as possible.
```

Comment:
We don't have CHANGELOG.md maybe point to official doc https://docs.nvidia.com/networking/software/cloud-orchestration/index.html